olp-cpp-sdk-dataservice-read/tests/CatalogClientTest.cpp

```diff
@@ -1630,13 +1630,16 @@ TEST_P(CatalogClientMockTest, DISABLED_GetPartitionsCancelLayerVersions) {
   CancelCallback cancel_mock;
 
   std::tie(request_id, send_mock, cancel_mock) = generateNetworkMocks(
-      waitForCancel, pauseForCancel, {200, HTTP_RESPONSE_LAYER_VERSIONS});
+      waitForCancel, pauseForCancel, {200, HTTP_RESPONSE_LOOKUP_CONFIG});
 
-  EXPECT_CALL(*network_mock_,
-              Send(IsGetRequest(URL_LAYER_VERSIONS), _, _, _, _))
+  EXPECT_CALL(*network_mock_, Send(IsGetRequest(URL_LOOKUP_CONFIG), _, _, _, _))
       .Times(1)
       .WillOnce(testing::Invoke(std::move(send_mock)));
 
+  EXPECT_CALL(*network_mock_,
+              Send(IsGetRequest(URL_LAYER_VERSIONS), _, _, _, _))
+      .Times(0);  // TODO: make test to wait/cancel at URL_LAYER_VERSIONS state
```

Comment:
As discussed, please have a look if the test case is still testing what it says: GetPartitionsCancelLayerVersions.
Currently it looks like we are cancelling look-up API call instead.

Reply:
cancel* tests fixes to enabled be in an upcoming pull-request